server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+var cors = require('cors')
+const app = express();
+
+app.use(cors())
+app.use(bodyParser.urlencoded({ extended: true }))
+
+app.use(bodyParser.json())
+
+app.get('/', (req, res) => {
+    res.json({ "message": "Welcome" });
+});
+
+app.get('/products', (req, res) => {
+    const limit = req.query.limit;
+    const page = req.query.page;
+    const offset = page * limit;
+    console.log(limit, page);
+    const obj = require("./src/data.json");
```

Comment:
Declare the const outside the Api and clone the same, currently the problem being faced is that the global object is being used, and once we splice, the global object gets affected.

possible solution approach is as follows : 
```
const productsMockData = require("./src/data.json");

```